topics/composer/index.md

```diff
@@ -1,4 +1,5 @@
 ---
+aliases: packagist,composer-package,packagist-package
```

Comment:
`packagist-package` is not currently used on any public repositories. Thoughts on removing it?
```suggestion
aliases: packagist,composer-package
```

Reply:
I added it to be more "consistant" with other package managers for other languages. Because for other package managers, there are both `PACKAGE-MANAGER-NAME-package` and `PACKAGE-INDEX-NAME-package`. Here, `packagist-package` is not used, by it may be in the future, and I don't see other reasons to remove it. But if there is any other thing about this, I can remove it if you want.

Reply:
Thanks for the context. No need to remove now, it's fine to stay I think.